k8s/manifests/charts/ck-loadbalancer/values.schema.json

```diff
@@ -22,10 +25,15 @@
       "type": "object",
       "properties": {
         "cidrs": {
-          "type": "array",
+          "type": [
+            "array"
+          ],
           "items": [
             {
-              "type": "object",
+              "type": [
+                "object",
+                "string"
+              ],
```

Comment:
no, do not make this either an object or a string. this will simply lead to confusion in the future, where people will be confused why the schema is sometimes an object and sometimes a string (and why would either be preferred).

we control the schema, we should pass objects here, just as we are already doing